src/components/List/List.jsx

```diff
@@ -18,15 +18,18 @@ const List = ({
   id = Rivet.shortuid(),
   className,
   children,
+  plain = false,
   variant = "unordered",
   ...attrs
 }) => {
   const classes = classNames(
     inline ? `${classPrefix}-inline` : classPrefix,
+    plain ? `${classPrefix}-plain` : "",
     variant === "plain" && `${classPrefix}-plain`,
-    className
+    variant === "description" && `${classPrefix}-description`,
+    className,
   );
-  const ListTag = variant === "ordered" ? "ol" : "ul";
+  const ListTag = getElement(variant);
```

Comment:
Rather than a function, you could do this:

```js
// At the top of the module:
const variantTagMap = {
  description: 'dl',
  ordered: 'ol',
  plain: 'ul',
  unordered: 'ul'
};

// Here:
const ListTag = variantTagMap[variant];
```

Up to you if you want to change it. But I feel like these mapping objects tend to be cleaner than functions and switch statements.

Reply:
Also, I wonder if we need to be a little more internally consistent on something. Here, we use both "tag" and "element" interchangeably. Then PR #511 uses the "component" prop. Maybe that's something we clean up at some point.

Reply:
Likewise, HTML conventions tends to use a "type" attribute similarly to how we're using the "variant" prop here. So again, I don't know if this is something to discuss.

Reply:
I agree with standardizing on `component` (I've seen that on other projects). HTML using `type` as an attribute throws a wrench in our library components.l I think it was alerts we had a problem with in the alpha and switched away from `type`.